apps/09todo-jquery/script.js

```diff
@@ -2,4 +2,19 @@
 
 $(document).ready(function() {
   // App logic goes here
+  $('form').submit(function (event) {
+    event.preventDefault();
+    var todoText = $(this).find('#todo').val();
+    $(this).find('#todo').val('');
+    $('#todo-list').append('<li>' + todoText + '<button></button>'+'</li>');
+    $('#todo-list').sortable();
```

Comment:
You can move line 10 to the top of the logic so it's not called every time a submit is performed.